packages/components/Callout/macos/CalloutWindow.swift

```diff
@@ -67,12 +67,13 @@ class CalloutWindow: NSWindow {
 
 	@objc private func dismissCallout() {
 		lifeCycleDelegate?.calloutWillDismiss(window: self)
-
+		orderOut(self)
```

Comment:
NSWindow.orderOut(_:) is  a well known and documented API, that you can get the documentation from apple easily by cmd+clicking on the method in Xcode. As such, I don't think  it needs to be documented here. 
https://developer.apple.com/documentation/appkit/nswindow/1419660-orderout

On whether the NSWindow object is released when closed, there's actually a separate prop for that: https://developer.apple.com/documentation/appkit/nswindow/1419062-releasedwhenclosed

I elected _not_ to release windows, and keep them around as long as the placeholder view `Callout` exists. This was mostly done because I kept on running to crashe / BAD_ACCESS errors if I _did_ release the window :') 

Apple did not make it have an easy built in transient window class I could use (at least, not one without a beak). So a lot of these decisions were made on the fly based on what I could get working. The decision _not_ to release when closed is one I could document somewhere.